nlp_apps.ipynb

```diff
@@ -797,11 +797,215 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {
     "collapsed": true
    },
+   "source": [
+    "## Text Analysis using TF-IDF\n",
+    "Since we know that computers are great with numbers but they cannot work out with the natural language. So in-order to overcome that text can be directly converted to numbers for analyzing.One of the most common and popular technique for this is TF-IDF which stands for Term Frequency and Inverse Document Frequency. \n",
```

Comment:
The first sentence is a bit awkward to read and not entirely grammatically correct. Remove the 'since' and try to restructure it a bit. In the second sentence, add a comma after 'that' and leave a space after the period. In the third sentence, it is 'techniques' instead of 'technique'.